rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1422,64 +770,6 @@ size_t MasterGraph::bounding_box_batch_count(int *buf, pMetaDataBatch meta_data_
     return size;
 }
 
-
-size_t MasterGraph::compute_optimum_internal_batch_size(size_t user_batch_size, RocalAffinity affinity)
```

Comment:
We need similar function to calculate OMP number of threads where user doesn't pass that

Reply:
We are working on the changes required to pass the threads from the user. We will add it in subsequent commits